Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNames.proto

```diff
@@ -9,97 +9,69 @@ import "google/api/resource.proto";
 import "google/api/field_behavior.proto";
 
 service ResourceNames {
-  option (google.api.default_host) = "resourcenames.example.com";
-
-  // Test single resource definition.
-  rpc SingleMethod(SingleResource) returns(Response) {
-    option (google.api.method_signature) = "name";
-  }
-
-  // Test single resource reference.
-  rpc SingleMethodRef(SingleResourceRef) returns(Response) {
-    option (google.api.method_signature) = "single_resource";
-  }
-
-  // Test multi resource definition.
-  rpc MultiMethod(MultiResource) returns(Response) {
-    option (google.api.method_signature) = "name";
-  }
-
-  // Test multi resource reference.
-  rpc MultiMethodRef(MultiResourceRef) returns(Response) {
-    option (google.api.method_signature) = "multi_resource";
+  rpc SinglePatternMethod(SinglePattern) returns(Response) {
+    option (google.api.method_signature) = "real_name, ref, repeated_ref";
   }
 
-  // Test future multi resource (FUTURE_MULTI_PATTERN).
-  rpc FutureMultiMethod(FutureMultiResource) returns(Response) {
-    option (google.api.method_signature) = "name";
+  rpc WildcardOnlyPatternMethod(WildcardOnlyPattern) returns(Response) {
+    option (google.api.method_signature) = "name, ref, repeated_ref, ref_sugar, repeated_ref_sugar";
   }
 
-  // Test original single resource (ORIGINALLY_SINGLE_PATTERN).
-  rpc OriginallySingleMethod(OriginallySingleResource) returns(Response) {
-    option (google.api.method_signature) = "name";
+  rpc WildcardMultiPatternMethod(WildcardMultiPattern) returns(Response) {
+    option (google.api.method_signature) = "name, ref, repeated_ref";
   }
 
-  // Test original single resource (ORIGINALLY_SINGLE_PATTERN),
-  // with combinatorial method generation.
-  rpc OriginallySingleMethodRef(OriginallySingleResourceRef) returns(Response) {
-    option (google.api.method_signature) = "resource1,resource2";
+  rpc WildcardMultiPatternMultipleMethod(WildcardMultiPatternMultiple) returns(Response) {
+    option (google.api.method_signature) = "ref, repeated_ref"; // `name` parameter delibrately missing.
   }
 }
 
-message SingleResource {
+message SinglePattern {
   option (google.api.resource) = {
-    type: "resourcenames.example.com/SingleResource"
+    type: "resourcenames.example.com/SinglePattern"
 	pattern: "items/{item_id}"
+    name_field: "real_name"
```

Comment:
Is there a reason why pattern is more indented than type and name_field?

Reply:
The whole tabs/spaces thing. Fixed.